frontend/src/BlogDetail/BlogDetail.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import NavBar from '../Navbar/NavBar';
+
+function BlogDetail() {
+	return (
+		<div>
+			<div>
+				<NavBar />
+			</div>
+			<div className='ui raised very padded text container'>
+				<div>
+					<h4 className='ui left aligned sub header'>Date Created</h4>
+					<h1 className='ui left aligned header'>Article Title</h1>
+				</div>
+				<div className='ui hidden divider'></div>
+				<div className='ui left aligned text container segment '>
+					<div className='ui placeholder'>
+						<div className='image header'>
+							<div className='line'></div>
+							<div className='line'></div>
+						</div>
+						<div className='paragraph'>
+							<div className='line'></div>
+							<div className='line'></div>
+							<div className='line'></div>
+							<div className='line'></div>
+							<div className='line'></div>
+						</div>
+					</div>
+				</div>
+
+				<div className='form container'>
+					<div className='ui form'>
+						<div className='field'>
+							<textarea> What are your thoughts?</textarea>
```

Comment:
```suggestion
							<textarea placeholder="What are your thoughts?" value=''></textarea>
```

You can make it a placeholder instead
![image](https://user-images.githubusercontent.com/108708225/183821058-06846c7e-056b-4fa5-825e-2127a29c608c.png)